docs/contrib/Demo_Mode.md

```diff
@@ -4,35 +4,35 @@ The kai server is able to cache results from a LLM and later replay these to aid
+When the server is run with `KAI__DEMO_MODE=true`, these responses will be played back. The request will be matched on everything except for authorization headers, cookies, content-length and request body.
 
-### `DEMO_MODE` Cached Responses
+### `KAI__DEMO_MODE` Cached Responses
 
 - We do not actively maintain cached responses for all models/requests.
 - You may look at: [kai/data/vcr/coolstore](kai/data/vcr/coolstore/) to see a list of what models have cached responses.
   - In general when we cache responses we are running: [example/run_demo.py](example/run_demo.py) and saving those responses.
     - This corresponds to a 'KAI Fix All' being run per file in Analysis.
 - When running from IDE and attempting to use cached response, we likely only have cached responses for 'Fix All', and we do not have cached responses for individual issues in a file.
 
-### `DEMO_MODE` Updating Cached Responses
+### `KAI__DEMO_MODE` Updating Cached Responses
 
 There are two ways to record new responses:
 
-1. Run the requests while the server is not in `DEMO_MODE`
+1. Run the requests while the server is not in `KAI__DEMO_MODE`
 1. Delete the specific existing cached response (under `kai/data/vcr/<application_name>/<model>/<source-file-path-with-slashes-replaced-with-dashes.java.yaml>`), then rerun. When a cached response does not exist, a new one will be recorded and played back on subsequent runs.
 
-### Seeing a `401` while running with `DEMO_MODE=TRUE`?
+### Seeing a `401` while running with `KAI__DEMO_MODE=TRUE`?
 
-If you are running with DEMO_MODE=TRUE and you see a `401` being returned from the LLM Provider consider that this is likely 'correct' behavior, if you do not have a valid API key defined. Note that DEMO_MODE will attempt to play back a cached response, yet if there is no cached data for the request Kai will attempt to talk to the LLM and will make a 'real' request, which if you don't have a valid API key will result in a `401`. To 'fix' this you will want to look at the request you are sending to Kai and ensure it is cached for the model you have configured, if you don't have valid cached data then you will need to get a valid api key and re-run so the data may be cached.
+If you are running with KAI**DEMO_MODE=TRUE and you see a `401` being returned from the LLM Provider consider that this is likely 'correct' behavior, if you do not have a valid API key defined. Note that KAI**DEMO_MODE will attempt to play back a cached response, yet if there is no cached data for the request Kai will attempt to talk to the LLM and will make a 'real' request, which if you don't have a valid API key will result in a `401`. To 'fix' this you will want to look at the request you are sending to Kai and ensure it is cached for the model you have configured, if you don't have valid cached data then you will need to get a valid api key and re-run so the data may be cached.
```

Comment:
Is `KAI**DEMO_MODE=TRUE` intended?

Reply:
No, I had an error with find and replace. Fixing.